.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "3rdparty/mshadow"]
 	path = 3rdparty/mshadow
-	url = https://github.com/dmlc/mshadow.git
+	url = https://github.com/apeforest/mshadow.git
```

Comment:
I see why you did this for testing, but do we want to check this is to master and depend on your fork?

Reply:
Yes, after mshadow PR is merged, I will update this here accordingly.

Reply:
https://github.com/dmlc/mshadow/pull/371 has been merged